test/cmd/diagnostics.sh

```diff
@@ -25,23 +25,30 @@ os::cmd::expect_failure_and_text "oc adm diagnostics NodeConfigCheck --node-conf
 
 os::cmd::expect_success 'oc adm policy reconcile-cluster-roles --additive-only=false --confirm'
 
-os::cmd::expect_success 'oc adm diagnostics ClusterRoleBindings ClusterRoles ConfigContexts '
+os::cmd::expect_success 'oc adm diagnostics ClusterRoleBindings '
+os::cmd::expect_success 'oc adm diagnostics ClusterRoles '
+os::cmd::expect_success 'oc adm diagnostics ConfigContexts '
 # DiagnosticPod can't run without Docker, would just time out. Exercise flags instead.
 os::cmd::expect_success "oc adm diagnostics DiagnosticPod --prevent-modification --images=foo"
-os::cmd::expect_success "oc adm diagnostics MasterConfigCheck NodeConfigCheck ServiceExternalIPs --master-config=${MASTER_CONFIG_DIR}/master-config.yaml --node-config=${NODE_CONFIG_DIR}/node-config.yaml"
+# EtcdWriteVolume can't run without etcd. Just exercise flags.
+os::cmd::expect_success "oc adm diagnostics EtcdWriteVolume --duration=10s --help"
+os::cmd::expect_success "oc adm diagnostics MasterConfigCheck --master-config=${MASTER_CONFIG_DIR}/master-config.yaml"
+os::cmd::expect_success "oc adm diagnostics NodeConfigCheck --node-config=${NODE_CONFIG_DIR}/node-config.yaml"
+os::cmd::expect_success "oc adm diagnostics ServiceExternalIPs --master-config=${MASTER_CONFIG_DIR}/master-config.yaml"
```

Comment:
nit: mind changing at least one of these to all lowercase to ensure aliases work?

Reply:
certainly